cli/commands/kms.go

```diff
@@ -25,16 +25,6 @@ var KMSCmds = cli.Command{
 			Before: createKeyValidator,
 			Action: createKeyHandler,
 		},
-		{
```

Comment:
keep it, just return the hdden secret on the back end. this will be useful for inspecting users in a key. @febg